lib/npg_qc/Schema/Result/AnalysisLane.pm

```diff
@@ -618,24 +618,9 @@ __PACKAGE__->belongs_to(
   { is_deferrable => 1, on_delete => 'NO ACTION', on_update => 'NO ACTION' },
 );
 
-=head2 analysis_lane_qcals
```

Comment:
Because we are updating the file we should probably update the year in the copyright legend.